ios/Wallet/characteristics/DisconnectCharacteristic.swift

```diff
@@ -0,0 +1,16 @@
+
+import Foundation
+import CoreBluetooth
+
+
+class DisconnectCharacteristic {
+  var data: Data
+  static let uuid = CharacteristicIds.DISCONNECT_CHAR_UUID
+  var status:  Int
+
+    init(data: Data){
+        self.data =  data
+        status = Int(data[0])
+         CRCValidator.verify(data: data, characteristic: DisconnectCharacteristic.uuid.rawValue)
```

Comment:
I now sort of understand why an error is thrown from CRCValidator, but have you tried to see if an error doesn't halt the transfer?